UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -9,8 +9,8 @@
 namespace HealthV2
 {
 	/// <summary>
-	/// A part of a body. Can be external, such as a limb, or internal like an organ.
-	/// Body parts can also contain other body parts, eg the 'brain body part' contained in the 'head body part'.
+	/// An external part of a body. Limbs, chest, head, wings, etc.
```

Comment:
this is incorrect, since organ still use BodyPart

Reply:
such as the game object heart and such

Reply:
reverted